lib/eslint/eslint.js

```diff
@@ -552,9 +552,12 @@ class ESLint {
             ...unknownOptions
         } = options || {};
 
-        for (const key of Object.keys(unknownOptions)) {
-            throw new Error(`'options' must not include the unknown option '${key}'`);
+        const unknownOptionKeys = Object.keys(unknownOptions);
+
+        if (unknownOptionKeys.length > 0) {
+            throw new Error(`'options' must not include the unknown option(s) '${unknownOptionKeys}'`);
```

Comment:
```suggestion
            throw new Error(`'options' must not include the unknown option(s): ${unknownOptionKeys.join(", ")}`);
```

This will be formatted like a similar error message in [processOptions](https://github.com/eslint/eslint/blob/e2bed2ead22b575d55ccaeed94eecd3a979dd871/lib/eslint/eslint.js#L183)